gravitee-am-common/src/main/java/io/gravitee/am/common/utils/ConstantKeys.java

```diff
@@ -76,7 +76,7 @@ public interface ConstantKeys {
 
     // MFA keys.
     String MFA_STOP = "mfaStop";
-    String MFA_CAN_BE_SKIPPED_KEY = "mfaEnrollmentCanBeSkipped";
+    String MFA_CAN_BE_CONDITIONAL_SKIPPED_KEY = "mfaEnrollmentCanBeSkipped";
```

Comment:
**TODO**: I don't understand why you renamed it to introduce conditional. Mfa can be skipped in the conditional or optional case

Reply:
It is used only for Conditional skip to remember the outcome of rule execution. For an optional case, this is not needed.